lib/ical/event.js

```diff
@@ -7,26 +7,36 @@ import { binsearchInsert } from "./helpers.js";
 import Component from "./component.js";
 import Property from "./property.js";
 import Timezone from "./timezone.js";
+// needed for typescript type resolution
+// eslint-disable-next-line no-unused-vars
+import Time from "./time.js";
+// eslint-disable-next-line no-unused-vars
+import Duration from "./duration.js";
 import RecurExpansion from "./recur_expansion.js";
 
+/**
+ * @ignore
```

Comment:
is this comment okay for you? While generating jsdoc it complains that `@ignore` does not permit a value but otherwise the description after ignore makes it into the jsdoc itself. Is the warning fine for you or should i move the description of `@ignore` up?

`WARNING: The @ignore tag does not permit a value; the value will be ignored. File: event.js, line: 17`

```ts
/**
 * This lets typescript resolve our custom types
 * (jsdoc typedefs are converted to typescript types)
 *
 * @ignore Ignore prevents the typedefs from being documented more than once
 * @typedef {import("./types.d.js").frequencyValues} frequencyValues
 * Imports the 'frequencyValues' type from the "types.d.js" module
 * @typedef {import("./types.d.js").occurrenceDetails} occurrenceDetails
 * Imports the 'occurrenceDetails' type from the "types.d.js" module
 */
```

Reply:
This is fine with me, thanks!